tests/test_main.py

```diff
@@ -41,7 +41,7 @@ def test_is_valid_position(self):
 
     def test_is_valid_position_invalid(self):
         invalid_positions = [
-            (2,3), (3,2), (3,3), (9,9), (-1,-1), 1, "something", False, (0,0,0)
+            (2,3), (3,2), (3,3), (9,9), (-1,-1), 1, "something", False, (0,0,0), ("st", 1)
         ]
         for position in invalid_positions:
             self.assertFalse(_position_is_valid(position))
```

Comment:
This is not the recommended way to write a test. Please remind me during the next meeting and I'll explain why. Don't worry too much about it now though, it does work and that's fine.